THIRD-PARTY-NOTICES.TXT

```diff
@@ -0,0 +1,82 @@
+ML.NET uses third-party libraries or other resources that may be
+
+distributed under licenses different than the ML.NET software.
```

Comment:
Why are there so many blank lines in this file?

Reply:
We use corefx's template. I don't think they are troublesome as you will see a lot of spaces in licenses from external softwares in this file.

Reply:
I got your point. Yeah, I will fix it.